packages/js/plugins/ipfs/package.json

```diff
@@ -12,12 +12,12 @@
     "build"
   ],
   "scripts": {
-    "build": "rimraf ./build && tsc --project tsconfig.build.json",
+    "build": "rimraf ./build && yarn codegen && tsc --project tsconfig.build.json",
     "codegen": "node ../../../../dependencies/node_modules/polywrap/bin/polywrap plugin codegen",
     "codegen:patch": "node ../../../cli/bin/polywrap plugin codegen && rimraf ./src/wrap",
     "lint": "eslint --color -c ../../../../.eslintrc.js src/",
     "test": "jest --passWithNoTests --runInBand --verbose",
-    "test:ci": "jest --passWithNoTests --runInBand --verbose",
```

Comment:
My initial build step here didn't have codegen, so tests would fail after build. Now that it has codegen, it's not required before running tests, you're right.